esmvaltool/preprocessor/_regrid.py

```diff
@@ -53,7 +54,7 @@
     unstructured_nearest=UnstructuredNearest())
 
 # Supported vertical interpolation schemes.
-vertical_schemes = ['linear', 'nearest']
+vertical_schemes = ['linear', 'nearest', 'nearest_extrap', 'linear_extrap']
```

Comment:
Use easy to read names, e.g. 'nearest_neighbour_extrapolation` instead of 'nearest_extrap'

Reply:
Went with the explicit naming scheme "linear_horizontal_extrapolate_vertical" to describe the new schemes.